### ISSUE TITLE
The color scheme is always detected incorrectly after initialization

### ISSUE THREAD

**Issue by foreverRuns** [slint-ui/slint#4198]
### The color scheme is always detected incorrectly after initialization.
1. start the application, the color scheme is detected correctly. (light mode)
2. change the windows to dark mode, the application detect light mode.
3. change the windows to light mode, the application detect dark mode.
   
```slint
import { StyleMetrics } from "std-widgets.slint";

export global Theme  {
  in-out property <brush> app-background: StyleMetrics.dark-color-scheme ? #243238 : #EEEEEE;
}
```

### LineEdit and TextEdit component can not use 'ctrl + v' to paste in caps lock mode

it can use in Slint: 1.2.2 version.

### Using

* OS: windows 11
* Language: Rust 1.73
* Slint: 1.3.2
* Slint-build : 1.3.2

**Comment by tronical** [comment]
Thank you for the report. Could you describe a little more in detail how you're changing the mode? I've tried changing it in the Windows 11 settings, and for example our gallery adjusts correctly, as you can see in the clip below:

https://github.com/slint-ui/slint/assets/1486/6e8168c5-3f2e-4888-98d0-b48e649d46fe

**Comment by foreverRuns** [comment]
Thanks for your reply. I change the mode as follows.
[20231221103527.webm](https://github.com/slint-ui/slint/assets/17795989/133af0ec-dc82-4b37-896d-fb6d5829b11f)

**Comment by foreverRuns** [comment]
Thanks for your reply. I change the mode as follows.

https://github.com/slint-ui/slint/assets/17795989/8e9c9a9b-e42d-4544-b4b8-3992959919fa

**Comment by ogoffart** [comment]
This is strange.
The dark/light detection comes from winit though. This might be a winit bug.

Could you try if the `theme` example from the winit reprository works correctly?


> LineEdit and TextEdit component can not use 'ctrl + v' to paste in caps lock mode

Reported a different issue for it to track it differently:  https://github.com/slint-ui/slint/issues/4221

**Comment by ogoffart** [comment]
I'm going to close this issue since it's not really actionable without further information.
Please comment with more information if you can still reproduce.
